e2e/env/common.go

```diff
@@ -53,6 +77,22 @@ func init() {
 		fmt.Printf("No value found in '%s': using default value %s\n", EnvVarTestedVersion, DefaultTestedVersion)
 	}
 
+	clusterIDPrefix = os.Getenv(EnvVarClusterIDPrefix)
+	if clusterIDPrefix == "" {
+		// Default cluster ID prefix is always the same for CI
+		clusterIDPrefix = DefaultClusterIDPrefix
+		fmt.Printf("No value found in '%s': using default value %s\n", EnvVarClusterIDPrefix, DefaultClusterIDPrefix)
+	}
+
+	if randomizeClusterID {
+		randomPrefixPart := entityid.New()
+		clusterIDPrefix = fmt.Sprintf("%s-%s", clusterIDPrefix, randomPrefixPart)
+	}
+
+	if randomizeClusterID && (len(clusterIDPrefix)+len(testedVersion) > MaxClusterIDPrefixAndTestedVersionLength) {
```

Comment:
It would be good to tell what the user can do to fix this, something like 
```
panic(fmt.Sprintf("Max length for cluster ID prefix (%s) + tested version (%s) version, when combined, is %d due to Azure resource name limitations (e.g. storage account name). Try passing a shorter prefix in '%s'", clusterIDPrefix, testedVersion, MaxClusterIDPrefixAndTestedVersionLength), EnvVarClusterIDPrefix)
```